### PR TITLE
added COC,made changes as per ansible collection requirements

### PR DESCRIPTION
**Pull request by ar-india**
Signed-off-by: ar-india <alokranjan2005@gmail.com>
https://github.com/ansible-collections/ansible-inclusion/discussions/13

### CODE REVIEW

File: ansible_collection/hpe/nimble/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. contents:: Topics
+
+v1.0.2
+======
+
+Release Summary
+---------------
+
+This release provides compatibilty for Ansible v4.0.0
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/CHANGELOG.rst]: compatibility spell check, compatibility with not for

File: ansible_collection/hpe/nimble/CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
+=================================================
+HPE Nimble Storage Ansible Modules Release Notes
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/CHANGELOG.rst]: HPE Nimble Storage Content Collection for Ansible

File: ansible_collection/hpe/nimble/.github/workflows/sanity_tests.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 6 * * *'
+env:
+  NAMESPACE: hpe
+  COLLECTION_NAME: nimble
+jobs:
+  sanity:
+    name: Sanity (Ⓐ${{ matrix.ansible }})
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/.github/workflows/sanity_tests.yml:12]: What kind of char is that?

File: ansible_collection/hpe/nimble/CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+- Made it compatible for Ansible v4.0.0
+
+v1.0.1
+======
+
+Release Summary
+---------------
+
+This release provides compatibilty for Ansible v3.0.0
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/CHANGELOG.rst]: compatibility spell check, compatibility with not for

File: ansible_collection/hpe/nimble/CHANGELOG.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Minor Changes
+-------------
+
+- Made it compatible for Ansible v3.0.0
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/CHANGELOG.rst]: compatible with

File: ansible_collection/hpe/nimble/CHANGELOG.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+Release Summary
+---------------
+
+This release provides compatibilty for Ansible v4.0.0
+
+Minor Changes
+-------------
+
+- Made it compatible for Ansible v4.0.0
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/CHANGELOG.rst]: compatible with

File: ansible_collection/hpe/nimble/CHANGELOG.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+v1.0.0
+======
+
+Release Summary
+---------------
+
+First release of the `HPE Nimble Storage Content Collection for Ansible`.
+
+New Modules
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/CHANGELOG.rst:41]: No folder still?!

File: ansible_collection/hpe/nimble/changelogs/changelog.yaml [hunk 8]
```diff
@@ -1,5 +1,15 @@
 ancestor: null
 releases:
+  1.0.2:
+    changes:
+      release_summary: 'This patch release provides compatibilty for Ansible v4.0.0'
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/changelogs/changelog.yaml]: compatibility spell check, compatibility with

File: ansible_collection/hpe/nimble/changelogs/changelog.yaml [hunk 9]
```diff
@@ -1,5 +1,15 @@
...
+    release_date: '2021-04-15'
+
+  1.0.1:
+    changes:
+      release_summary: 'This patch release provides compatibilty for Ansible v3.0.0'
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/changelogs/changelog.yaml]: compatibility with

**Review by datamattsson — CHANGES_REQUESTED** [review]
Small nits.

File: ansible_collection/hpe/nimble/CHANGELOG.rst (hunk 7 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/CHANGELOG.rst:41]: yes as we discussed this release is still with no code changes.. implementing folder will take time and we will surely miss the deadline. plan was to take it in next release.

File: ansible_collection/hpe/nimble/.github/workflows/sanity_tests.yml (hunk 3 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/.github/workflows/sanity_tests.yml:12]: most likely represents ansible.. have just picked that from ansible documented sanity test shared by ansible reviewer

https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml

**Review by suneeth51 — APPROVED** [review]
lgtm..

**Review by datamattsson — APPROVED** [review]
LGTM

**Pull request merged**